.github/workflows/check-compatibility.yml

```diff
@@ -0,0 +1,42 @@
+name: "Check compatibility"
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-compatibility:
+    name: 'Check compatibility'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node: [18, 20, 22]
+        eslint: [8, 9]
```

Comment:
8 is (already) EOL, I don't think it makes sense to have it in here

Reply:
Fair enough! I'd like to leave the CI-structure in place though (eventhough ESLint is only allowed to have one value); I think it might come in handy in the future and it doesn't do harm for now.